Packs/WhatIsMyBrowser/Integrations/WhatIsMyBrowser/WhatIsMyBrowser.py

```
@@ -19,8 +20,11 @@
 
 
 def http_request(data):
+    api_key = demisto.params().get('credentials_api_key', {}).get('password') or demisto.params().get('api_key')
+    if not api_key:
+        return_error('API key must be provided.')
```

Raise exception instead of return_error